internal/clients/logger/github_logger/github_logger.go

```diff
@@ -39,6 +39,11 @@ func (l *GithubLogger) Debug(msg string) {
 	l.log.Printf("%s%s\n", debugPrefix, msg)
 }
 
+// Info logs an info message
+func (l *GithubLogger) Info(msg string) {
+	l.log.Println(msg)
```

Comment:
I am wondering if we want any prefix here. I cannot think of any good prefix, looking at other actions they also just plainly log text so probably fine.